<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Export</class>
 <widget class="QDialog" name="Export">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" colspan="4">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Zapisz</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="SciezkaDostepu">
       <property name="text">
        <string>Ścieżka dostepu</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="FormatZapisu">
       <property name="text">
        <string>Format zapisu</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="NazwaPliku">
       <property name="text">
        <string>Nazwa pliku</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QComboBox" name="comboBox">
       <property name="placeholderText">
        <string>format zapisu</string>
       </property>
       <item>
        <property name="text">
         <string>.CSV</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QLineEdit" name="SciezkaDostepu_LineEdit">
       <property name="text">
        <string>C:\Users</string>
       </property>
       <property name="placeholderText">
        <string>ścieżka dostępu</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="NazwaPliku_LineEdit">
       <property name="text">
        <string>Plik</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
